### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by denizzerdenn** [zjc062/mind-vis#18]
Hey there!! I loved your work, amazing. 

I'm trying to use this model to reconstruct images from a different fMRI data that I have but I have a time restriction so to do all the pre-training from scratch will result in a lot of time loss for me. I was wondering if you can share the corresponding files with me so that I can continue with the pre-trained model. 

Thank you

**Comment by shanibenamitay** [comment]
Hi @denizzerdenn  did you manage to finetune the model on your data?Would be happy to here how did you preprocess it?
